### PR TITLE
Delete dashboard docs added

### PR DESCRIPTION
**Pull request by pablopareja**

### CODE REVIEW

File: source/includes/api/_dashboard.md [hunk 1]
```diff
@@ -233,13 +233,13 @@ In order to perform this operation, the following conditions must be met:
 - the user must be logged in and belong to the same application as the dashboard
 - the user must match one of the following:
   - have role `ADMIN`
-  - have role `MANAGER` and be the dashboard's owner (through the `user-id` field of the dashboard)
+  - have role `MANAGER` or `USER` and be the dashboard's owner (through the `user-id` field of the dashboard)
```
Comment (tiagojsag) [inline source/includes/api/_dashboard.md]: https://github.com/resource-watch/resource-watch-manager/blob/35c71d09305c2d778ed49e07f04d46f324f9d937/spec/controllers/api/dashboards_update_spec.rb#L43

This was modified recently to mimic the behavior already found on other similar microservices - USERs cannot edit resources (even their own)

File: source/includes/api/_dashboard.md [hunk 2]
```diff
@@ -281,7 +281,21 @@ curl -X PATCH https://api.resourcewatch.org/v1/dashboards/<id of the dashboard>
...
 ```
  
+## Delete dashboard
 
+In order to perform this operation, the following conditions must be met:
+
+- the user must be logged in and belong to the same application as the dashboard
+- the user must match one of the following:
+  - have role `ADMIN`
+  - have role `MANAGER` or `USER` and be the dashboard's owner (through the `user-id` field of the dashboard)
```
Comment (tiagojsag) [inline source/includes/api/_dashboard.md]: I think, technically, right now, any authenticates user can delete a dashboard. However, this is probably due to an oversight, and I think we should aim to change this so it mimics other resources (datasets, widgets, etc) where admins can delete any, managers their own, and users none

**Review by tiagojsag — CHANGES_REQUESTED** [review]

**Comment by tiagojsag** [comment]
https://www.pivotaltracker.com/story/show/170050085

**Comment by pablopareja** [comment]
@tiagojsag the changes you requested have been included in my last commit 😉

**Pull request merged**